### DISCUSSION TITLE
MonitorControl preventing Mac/display sleep?

### DISCUSSION THREAD

**Discussion by dtsimon** [MonitorControl/MonitorControl#1642] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

MonitorControl appears to be preventing my external monitor (and my connected MBP) from sleeping. I turned off MonitorControl and the monitor and Mac correctly went to sleep.

I've tested on two identical monitors, both connected through OWC Thunderbolt docks.

### Expected behavior

The monitor and Mac should go to sleep according to the Mac energy settings.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.2.1
- Mac model: 13-inch, 2019, Four Thunderbolt 3 ports
- MonitorControl version: 4.0.2
- Monitor(s): Dell U3818DW (firmware M3B106)
- Monitor Cable(s)/Connection(s): Thunderbolt, through OWC Thunderbolt dock
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
I don't see how could MonitorControl prevent the mac from sleeping. You can check out what is preventing sleep by looking at the Activity Manager's Energy tab and see the `Preventing Sleep` column.

**Comment by waydabber** [comment]
Hey @dtsimon - let me know if you see anything - does the app prevent sleep according to the Activity Manager? Should I keep this issue open? Thank you!

**Comment by dtsimon** [comment]
Hi Istvan!

I’m traveling this week, so I can’t test. I’ll let you know next week.

/Daniel

**Comment by macOS-Mavericks** [comment]
I'm experiencing the same exact issue with Version 4.0.2 Build 6965 running on a M1 with macOS Big Sur 11.6 . I must remember to slide the control to full brightness so that sleep will work. As mentioned above, the Aerial screensaver seems to have found a temporary workaround for perhaps the same bug - https://github.com/JohnCoates/Aerial/releases/tag/v3.0.8beta1 .

**Comment by ValXp** [comment]
Same thing here for version 4.1.0 Build 7034. On an M1 16in MBP running Ventura 13.0.1.
Easy steps to verify:
- Put the computer to sleep and swipe on the trackpad, the computer wakes because it's not sleeping.
- Quit MonitorControl and do the previous step again. Swiping the trackpad won't wake the computer because it's actually sleeping.

**Comment by DasUBrot** [comment]
Macbook Air M1 8/256 with Sonoma 14.1.2 (23B92), current version of MonitorControl as of 2023/12: 
Same problem here. Macbook either does not go to sleep or wakes up a few minutes later when MonitorControl is running. Everything working fine when MonitorControl is closed.
A real shame because the program is simple to use, good-looking and really useful.

**Comment by waydabber** [comment]
Not sure whether this is still an issue. You can check what app prevents sleeping in Activity Monitor or by entering this in Terminal:

`pmset -g assertions`

Normally the app should not prevent sleep (there is nothing in it that should interfere afaik).
